src/components/editor/EditorAreaMenu.tsx

```diff
@@ -141,10 +153,7 @@ export default function Banner(props: { [x: string]: any }) {
 						bg: 'transparent'
 					}}
 					mb='10px'
-					onClick={() => {
-						const editorContent = document.getElementById("editor-main")?.innerText;
-						handleSave(editorContent);
-					  }}
+					onClick={(handleSave)}
```

Comment:
are sure about it?

Reply:
I'm not sure what you're asking @1-Harshit. I cleared out the lines I wrote inside the props of both handleSave and handleHTMLSave.

Reply:
```suggestion
					onClick={handleSave}
```

Reply:
God, I'm sorry. I simply couldn't see it 😂